src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -482,7 +482,7 @@ public function getWriteInfo(string $class, string $property, array $context = [
         if ($allowMagicSet) {
             [$accessible, $methodAccessibleErrors] = $this->isMethodAccessible($reflClass, '__set', 2);
             if ($accessible) {
-                return new PropertyWriteInfo(PropertyWriteInfo::TYPE_PROPERTY, $property, PropertyWriteInfo::VISIBILITY_PUBLIC, false);
+                return new PropertyWriteInfo(PropertyWriteInfo::TYPE_PROPERTY, $property, PropertyWriteInfo::VISIBILITY_PUBLIC, false, false);
```

Comment:
Same here?

Reply:
For this one we only get here if the property doesn't exists (case handled at rows 472-480) so we know for sure there is no property at this point.